workitem/workitem_repository.go

```diff
@@ -314,6 +317,37 @@ func (r *GormWorkItemRepository) List(ctx context.Context, criteria criteria.Exp
 	return res, count, nil
 }
 
+// ListChildren get all child work items
+func (r *GormWorkItemRepository) ListChildren(ctx context.Context, parent string) ([]*app.WorkItem, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "workitem", "children", "query"}, time.Now())
+
+	where := "id in (select work_item_links.target_id from work_item_links left join work_item_link_types on work_item_links.link_type_id = work_item_link_types.id where work_item_link_types.forward_name='relates to' and work_item_links.source_id = ?)"
```

Comment:
@aslakknutsen is right.

```sql
id in (
    SELECT work_item_links.target_id
    FROM work_item_links
    LEFT JOIN work_item_link_types ON work_item_links.link_type_id = work_item_link_types.id
    WHERE
        work_item_link_types.forward_name='relates to'
        AND work_item_links.source_id = ?)
```

When I formatted this I noticed a few things.

1. You should replace the hard coded table names with something like `workitem.WorkItemType{}.TableName()`
1. Instead of doing a `LEFT JOIN` in SQL I would query for the correct link type once and then just filter by ID:
    ```go
    fmt.Sprintf("id in (SELECT id FROM %[1]s WHERE source_id=? AND type_id=?)", link.WorkItemLinks{}.TableName())
    ```
    This should be much much faster than the `LEFT JOIN`.